website/contributing/release-roles-responsibilites.md

```diff
@@ -88,47 +86,4 @@ Two sub-roles:
-
-No strict coupling with any specific release - active supports and testers will be thanked in the release notes of versions they help with.
-
-### Release Supporter Responsibilities
-
-- Surface release issues either on stable or release candidate by triaging [release issues in the react-native repo](https://github.com/facebook/react-native/labels?q=release)
-  - Some issues might not be tagged appropriately, so keep an eye out on incoming issues and surface any
-- Watch the [release discussions repo](https://github.com/reactwg/react-native-releases/discussions)
-  - Help answer/debug/escalate issues
-- Work on any [release improvements](https://github.com/facebook/react-native/projects/18) or if you see something that can be improved; please add!
-- Help [test release candidates](/contributing/release-testing) with your configuration or improve it
-- Engage/help out with discussion in the release related channels (#supporters-feed, #testers-feedback, #release-coordination)
-
-### Release Tester Responsibilities
-
-This role is about helping test release candidates against your production app/workflows
-
-- Helps surface release issues either on stable or release candidate by them testing out against their production apps and workflows
-  - Perhaps integrate either the [npm `next` or `nightly` versions of react-native](https://www.npmjs.com/package/react-native) in your app's CI and raise any issues that might come up.
-- Support regression fixes if relevant
-- Engage/help out with discussion in the release related channels (#supporters-feed, #testers-feedback, #release-coordination)
-
-### Who can fill it
-
-- Anyone interested in supporting the React Native Open Source project and its releases!
-  - To start, you can participate in the [discussion repo](https://github.com/reactwg/react-native-releases/discussions) -- testing release candidates, surfacing any release issues you've seen or encountered yourself
-  - We also have some [discussions about improvements](https://github.com/reactwg/react-native-releases/discussions/categories/improvements) here as well as some issues related to [improving the release process in this project board](https://github.com/facebook/react-native/projects/18).
-    - If you are up for any specific tasks, let us know by commenting on it.
-- For release testers, it's preferred (as it's very valuable) that you are able to test the releases against a production app in order to also verify non-trivial parts of the flow like archiving a release for the App Store.
-
-#### Apply to the role
-
-For both supporters and testers, you can let us know that you want to help out in this [dedicated discussion](https://github.com/reactwg/react-native-releases/discussions/11). We will provide you access to a dedicated RN Discord server that folks involved the releasers use to coordinate.
+- Engineers from key companies in the React Native ecosystem are preferred.
```

Comment:
Maybe we should instead say -- this is a requirement of being a partner in the ecosystem and link to the ecosystem doc?

Reply:
Maybe that's something that we can do after we have an actual "this is what a partner is" type of doc? We touched on that idea during the last meeting but didn't agree nor finalized anything, as far as I know.

Would be great to revisit this once we have more clarity around partners and a new ecosystem doc

Reply:
Sounds good, I think then does it make sense to add this? I think it's alienating right now and not super useful. Once we get the ecosystem updates out (I'll put up a PR for this today), then I htink saying this is a bit clearer about what 'key companies' mean

Reply:
sounds good, I'll remove the line